Gemfile.lock

```diff
@@ -1,62 +1,60 @@
 GEM
   remote: https://rubygems.org/
   specs:
-    better_errors (2.1.1)
+    better_errors (2.5.0)
       coderay (>= 1.0.0)
-      erubis (>= 2.6.6)
+      erubi (>= 1.0.0)
       rack (>= 0.9.0)
-    byebug (9.0.6)
-    coderay (1.1.1)
-    commander (4.4.3)
-      highline (~> 1.7.2)
-    database_cleaner (1.6.1)
+    byebug (10.0.2)
+    coderay (1.1.2)
+    commander (4.4.7)
+      highline (~> 2.0.0)
+    database_cleaner (1.7.0)
     diff-lcs (1.3)
-    erubis (2.7.0)
-    event_sourcery (0.13.0)
-    event_sourcery-postgres (0.3.0)
-      event_sourcery (>= 0.10.0)
+    erubi (1.7.1)
+    event_sourcery (0.22.0)
+    event_sourcery-postgres (0.8.0)
+      event_sourcery (>= 0.14.0)
       pg
-      sequel (~> 4.38)
-    highline (1.7.8)
-    method_source (0.8.2)
-    mustermann (1.0.0)
+      sequel (>= 4.38)
+    highline (2.0.0)
+    method_source (0.9.2)
+    mustermann (1.0.3)
     pg (0.20.0)
-    pry (0.10.4)
+    pry (0.12.2)
       coderay (~> 1.1.0)
-      method_source (~> 0.8.1)
-      slop (~> 3.4)
-    pry-byebug (3.4.2)
-      byebug (~> 9.0)
+      method_source (~> 0.9.0)
+    pry-byebug (3.6.0)
+      byebug (~> 10.0)
       pry (~> 0.10)
-    rack (2.0.3)
-    rack-protection (2.0.0)
+    rack (2.0.6)
```

Comment:
### [rack](https://rack.github.io/)

**Patch** version upgrade :chart_with_upwards_trend::small_blue_diamond: 2.0.3 → 2.0.6

[_~~change-log~~, [source-code](http://github.com/rack/rack)_]

:tada: Patched vulnerabilities:

- [CVE-2018-16471](https://nvd.nist.gov/vuln/detail/CVE-2018-16471)
  **Possible XSS vulnerability in Rack**

  
  URL: https://groups.google.com/forum/#!topic/ruby-security-ann/NAalCee8n6o

  There is a possible vulnerability in Rack. This vulnerability has been assigned the CVE identifier CVE-2018-16471.  Versions Affected:  All. Not affected:       None. Fixed Versions:     2.0.6, 1.6.11  Impact ------ There is a possible XSS vulnerability in Rack.  Carefully crafted requests can impact the data returned by the `scheme` method on `Rack::Request`. Applications that expect the scheme to be limited to "http" or "https" and do not escape the return value could be vulnerable to an XSS attack.  Vulnerable code looks something like this:  ``` <%= request.scheme.html_safe %> ```  Note that applications using the normal escaping mechanisms provided by Rails may not impacted, but applications that bypass the escaping mechanisms, or do not use them may be vulnerable.  All users running an affected release should either upgrade or use one of the workarounds immediately.  Releases -------- The 2.0.6 and 1.6.11 releases are available at the normal locations.  Workarounds ----------- The following monkey patch can be applied to work around this issue:  ``` require "rack" require "rack/request"  class Rack::Request SCHEME_WHITELIST = %w(https http).freeze  def scheme   if get_header(Rack::HTTPS) == 'on'     'https'   elsif get_header(HTTP_X_FORWARDED_SSL) == 'on'     'https'   elsif forwarded_scheme     forwarded_scheme   else     get_header(Rack::RACK_URL_SCHEME)   end end  def forwarded_scheme   scheme_headers = [     get_header(HTTP_X_FORWARDED_SCHEME),     get_header(HTTP_X_FORWARDED_PROTO).to_s.split(',')[0]   ]    scheme_headers.each do |header|     return header if SCHEME_WHITELIST.include?(header)   end    nil end end ```






<details>
<summary>Commits</summary>

A change of **19** commits. See the full changes on [the compare page](https://github.com/rack/rack/compare/2.0.3...2.0.6).

These are the first 10 commits:
- (b5a351c) [Require the right file for the digest we're using](https://github.com/rack/rack/commit/b5a351cd92db3197f6186245e596013a520fcdc4)
- (0945c09) [Merge pull request #1189 from lugray/fix_rack_lock](https://github.com/rack/rack/commit/0945c094d12dfc6c2df73987570b53adc3f53f03)
- (f6c8bf1) [Merge pull request #1151 from cremno/simplify-some-string-cr…](https://github.com/rack/rack/commit/f6c8bf13befba29d88ea7df93c103344c31388da)
- (6dca575) [Merge pull request #1179 from tompng/master](https://github.com/rack/rack/commit/6dca575de1718e011d40bedfed425d4c7eed16f4)
- (c43217a) [Merge pull request #1192 from jkowens/master](https://github.com/rack/rack/commit/c43217a81917de03aa6ceb1aa485ae69b8bb4598)
- (d0fc020) [Merge pull request #1193 from tompng/multipart_less_memory](https://github.com/rack/rack/commit/d0fc020f7a1a9c6c04b33ed0644d310d76f7c74c)
- (c1f1d31) [Merge pull request #1190 from hugoabonizio/master](https://github.com/rack/rack/commit/c1f1d3176d3b9fe080a62a168b47dc7baef3fa7e)
- (2618257) [webrick: remove concurrent-ruby dev dependency](https://github.com/rack/rack/commit/261825768e3e769959f3eb30ec9f24a707a9b2ac)
- (0a95875) [Bumping version](https://github.com/rack/rack/commit/0a95875745ec65e91a57460a41373ae4d3a94934)
- (d4283e4) [Leahize](https://github.com/rack/rack/commit/d4283e4fb14b7d9f5f0ef7bb90d36476a9a6ece0)


</details>

Reply:
*This* was the reason for the `bundle update`.